api/loader/fileloader_test.go

```diff
@@ -201,23 +206,44 @@ func TestNewEmptyLoader(t *testing.T) {
 	require.Error(t, err)
 }
 
-func TestNewLoaderHTTP(t *testing.T) {
-	t.Run("doesn't exist", func(t *testing.T) {
-		_, err := makeLoader().New("https://google.com/project")
-		require.Error(t, err)
+func TestNewRemoteLoaderDoesNotExist(t *testing.T) {
+	_, err := makeLoader().New("https://example.com/project")
```

Comment:
I think it will be easy to understand to use the same URL for getting the same reason.
 https://github.com/kubernetes-sigs/kustomize/pull/5225/files#diff-15a9eb3d3d62fd27ecea240ff711d887220b80ac54f3f709a7ed8bc25632a837R224.

Reply:
Really great catch! I've changed the root url.

I did a little digging, and it turns out I blindly copied this existing test that was running incorrectly. The history of this test is as follows:
1. This test existed before [my previous PR](https://github.com/kubernetes-sigs/kustomize/commit/168e31bfb6733ebbd7ec562d6bd223790aa09b43#diff-f036a91524edc0f1e5eae1765348f9b6a9d79bfa43200296010714ce1a4e97f2L204-R207), which only moved it. 
2. This test existed at a time when [any `http` url was considered a repo](https://github.com/kubernetes-sigs/kustomize/blob/885c1952a47d6a19419311958ecbf864275374f0/pkg/loader/githubloader.go#L84)

Hence, this test originally wanted to check that `New()` failed when the repo url didn't exist. I assume someone forgot to update this test when they updated `api/internal/git/repospec`.

To defend against this, I check in the error that the loader tried to `fetch` the url. 

The url is not exactly the same as in the `accumulation_test` because there  I want to clarify where a url is treated as a remote file vs. repo, whereas I don't think we need that here since we explicitly call `New()`. I'm also happy to change this on further feedback, though.

Reply:
Thanks for investigating!